tests/test_install.py

```diff
@@ -36,21 +38,26 @@
         ),
     ],
 )
-def test(yaml_s, package, module):
+def test(tmpdir, yaml_s, package, module):
+
+    path = os.path.join(tmpdir, 'tmp.yaml')
+    with open(path, 'w') as fp:
+        fp.write(yaml_s)
+
     # fails because of missing packages
     with pytest.raises(ModuleNotFoundError):
-        audobject.from_yaml_s(
-            yaml_s,
+        audobject.from_yaml(
```

Comment:
`from_yaml_s` is now automatically covered by `from_yaml()` correct?

Reply:
I looked at the code and this is not the case, e.g. https://github.com/audeering/audobject/blob/master/audobject/core/api.py#L55-L83

So I guess we need ttest for `from_yaml_s()` as well?

Reply:
Now tests first `from_yaml_s()`, uninstalls package and repeats for `from_yaml()`. Hopefully, we now everything covered :)

Reply:
Had a final close look and actually there are more places where we do not pass on the `auto_install` argument. Will add some more tests :)

Reply:
I created https://github.com/audeering/audonnx/pull/15 and tested it locally with this branch.

Reply:
... and it worked :)

Reply:
Removed the WIP again.